Error of izpack-maven-plugin for recent versions of openjdk 21

For recent openjdk 21, izpack fails to package LSC:
- openjdk version "21.0.2" 2024-01-16 is working
- openjdk version "21.0.6" 2025-01-21 is failing
- openjdk version "21.0.7-ea" 2025-04-15 is failing

Error message:

```
[ERROR] Failed to execute goal org.codehaus.izpack:izpack-maven-plugin:5.2.3:izpack (standard-installer) on project lsc-core: Execution standard-installer of goal org.codehaus.izpack:izpack-maven-plugin:5.2.3:izpack failed: java.lang.ArrayIndexOutOfBoundsException: Index 70131 out of bounds for length 22674 -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -252,7 +252,7 @@
 			<plugin>
 				<groupId>org.codehaus.izpack</groupId>
 				<artifactId>izpack-maven-plugin</artifactId>
-				<version>5.2.3</version>
+				<version>5.2.4</version>
 
 				<!-- common configuration by all executions -->
 				<configuration>

